test/typecheck.jl

```diff
@@ -148,7 +148,7 @@ end
 msgs = lintstr(s)
 @test msgs[1].code == :E521
 @test msgs[1].variable == "a"
-@test contains(msgs[1].message, "apparent type DataType is not a container type")
+@test contains(msgs[1].message, "apparent type Type")
```

Comment:
The error message is actually the same. I had to change the test because it prints as `Type{T}` on 0.5 or `Type` on 0.6, and I didn't want to select which based off version.